sr_moveit_planner_benchmarking/experiments/launch/load_all_scenes_and_queries_to_db.launch

```diff
@@ -0,0 +1,54 @@
+<launch>
+    <!-- Load robot and warehouse-->
```

Comment:
should we cange all the "default" to "value" if we dont parametrize from outside? seems a bit insonsistent when some of them are "value" and some "defualt"

Reply:
Yes please change

Reply:
Yes please change